sympy/stats/tests/test_joint_rv.py

```diff
@@ -46,6 +48,38 @@ def test_Normal():
     # assert E(X) == mu
     # assert variance(X) == sigma
 
+    # test symbolic multivariate normal densities
+    n = 3
+
+    Sg = MatrixSymbol('Sg', n, n)
+    mu = MatrixSymbol('mu', n, 1)
+    obs = MatrixSymbol('obs', n, 1)
+
+    X = MultivariateNormal('X', mu, Sg)
+    density_X = density(X)
+
+    eval_a = density_X(obs).subs({Sg: eye(3),
+        mu: Matrix([0, 0, 0]), obs: Matrix([0, 0, 0])}).doit()
+    eval_b = density_X(0, 0, 0).subs({Sg: eye(3), mu: Matrix([0, 0, 0])}).doit()
```

Comment:
I would have expected that the `density_X(0, 0, 0)` would need to be `density_X(Matrix([0, 0, 0]))` or something. Does this convert the three separate positional arguments into a matrix somehow?

Reply:
I could imagine `density_X([0, 0, 0])` being accepted but it seems odd to allow multiple positional arguments in place of a single positional argument.

Reply:
This is the way the base code has been written - I haven't changed it. I also agree, having one positional argument is probably better, but that might break downstream code...

Reply:
So, this [line in master](https://github.com/sympy/sympy/blob/master/sympy/stats/joint_rv_types.py#L146) takes in `*args` as the input, and forces a conversion of the list `args` to an `ImmutableMatrix`.

Reply:
Oh, I see. The changes here make it possible to pass a single matrix in instead of multiple args.

Reply:
Yep

Reply:
Are there examples using multiple args like this in the docs? If so maybe they could be updated to pass a single matrix as you've done here.

Reply:
I have added an example in the docs about passing a matrix as an input.